frontend/demo/component/treegrid/treegrid-column.ts

```diff
@@ -0,0 +1,81 @@
+import '../../init'; // hidden-full-source-line
+import '@vaadin/flow-frontend/gridConnector.js'; // hidden-full-source-line (Grid's connector)
+
+import { customElement, LitElement, query } from 'lit-element';
+import { html } from 'lit-html';
+import '@vaadin/vaadin-grid/vaadin-grid';
+import '@vaadin/vaadin-grid/vaadin-grid-tree-column';
+import '@vaadin/vaadin-ordered-layout/vaadin-horizontal-layout';
+import '@vaadin/vaadin-button/vaadin-button';
+import {
+  GridDataProviderCallback,
+  GridDataProviderParams,
+  GridElement
+} from '@vaadin/vaadin-grid/vaadin-grid';
+import { getPeople } from '../../domain/DataService';
+import Person from '../../../generated/com/vaadin/demo/domain/Person';
+import { applyTheme } from 'generated/theme';
+
+@customElement('treegrid-column')
+export class Example extends LitElement {
+  constructor() {
+    super();
+    // Apply custom theme (only supported if your app uses one)
+    applyTheme(this.shadowRoot);
+  }
+
+  private managers?: Person[];
+
+  // async firstUpdated() {
+  //   this.managers = (await getPeople({managerId: null}));
+  // }
+
+  async dataProvider(params: GridDataProviderParams, callback: GridDataProviderCallback) {
```

Comment:
Using a bind in a template is discouraged (by eslint-plugin-lit). You can avoid having to `bind(this)` the data provider function by making this a class property
```suggestion
  private dataProvider = async (
    params: GridDataProviderParams,
    callback: GridDataProviderCallback
  ) => {
```

Reply:
Refactored to use a class property instead.